client/src/components/MobileAd/mobile-ad-modal.tsx

```diff
@@ -0,0 +1,58 @@
+import { Modal } from '@freecodecamp/react-bootstrap';
+import Bowser from 'bowser';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import './mobile-ad-modal.css';
+
+function MobileAdShow(): JSX.Element {
+  const [show, setShow] = useState(true);
+  const os = Bowser.getParser(window.navigator.userAgent).getOSName();
+  const { t } = useTranslation();
+
+  function close() {
+    if (typeof window !== 'undefined') {
+      localStorage.setItem('firstTime', 'true');
+    }
+    setShow(false);
+  }
+  return (
+    <>
+      {(os === 'iOS' || os === 'Android') && (
```

Comment:
To avoid confusion, we should only add a CTA to download an app for Android devices at this time - the app is not available on iOS yet, so iOS users should only see the link to the article.